### PR TITLE
Improve collectInfo command description

### PR DESCRIPTION
**Pull request by jiacheliu3**
This separates the code change from doc change in favor of https://github.com/Alluxio/alluxio/pull/10926/files

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/bundler/CollectInfo.java [hunk 1]
```diff
@@ -60,6 +61,20 @@
...
+                  + "about your Alluxio cluster. In the end of the run, "
+                  + "the collected information will be written to files "
+                  + "and bundled into one tarball.\n"
+                  + "[--max-threads <threadNum>] controlls how many threads this command uses. "
+                  + "By default it allocates one thread for each host."
+                  + "Use a smaller number to constrain the network IO when transmitting tarballs.\n"
+                  + "[--local] specifies this command should only collect "
+                  + "information about the localhost.\n"
+                  + "NOTE: You should ALWAYS check the credential risk about this command at\n"
+                  + "https://docs.alluxio.io/os/user/edge/en/operation/Troubleshooting.html"
```
Comment (jiacheliu3) [inline shell/src/main/java/alluxio/cli/bundler/CollectInfo.java]: Left the link to doc page, do you think the users will follow the link?

Comment (ns1123) [reply shell/src/main/java/alluxio/cli/bundler/CollectInfo.java]: I think you should explicitly put a message saying "This command MAY bundle credentials. To understand the risks refer to the docs here."

And ideally you can link to a specific subsection of the page not the main page, but that's more a nice to have.

Comment (jiacheliu3) [reply shell/src/main/java/alluxio/cli/bundler/CollectInfo.java]: Adjusted the message to your advice. Also changed `NOTE` to `WARNING`.

**Comment by ns1123** [comment]
LGTM

**Comment by jiacheliu3** [comment]
@ZacBlanco this is good to go. Do you mind merging this PR? Thanks!

**Comment by jiacheliu3** [comment]
alluxio-bot, test this please

**Comment by jiacheliu3** [comment]
Closed as this bases off 2.2, which is an incorrect approach.
Created https://github.com/Alluxio/alluxio/pull/11037 in place of this PR. Please merge #11037 instead. Thank you!

**Pull request closed without merging**